vars/fhcapProviderConfigFor.groovy

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/groovy
+
+def call(providerId, providers, template = 'default') {
+    def providerCfg = null
```

Comment:
Is this a Map? Do you ever expect it to be anything else? Is it ever going to be anything but `Map<String, String>`?